tests/functional/fx_ios_v1_sign_in.js

```diff
@@ -142,6 +142,26 @@ define([
           .then(testElementExists(selectors.SIGNUP_COMPLETE.HEADER));
     },
 
+    'Fx iOS >= 11.0 unverified, verify same browser': function () {
```

Comment:
This test doesn't really do anything different to the Fx iOS >= 6.1 test, might as well drop it.